src/main/java/core/basesyntax/service/impl/ReportGeneratorImpl.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.dao.FruitDao;
+import core.basesyntax.service.ReportGenerator;
+import java.util.stream.Collectors;
+
+public class ReportGeneratorImpl implements ReportGenerator {
+    private static final String reportHead = "fruit,quantity";
+    private static final String reportDataDelimiter = ",";
```

Comment:
The constant 'reportDataDelimiter' should also follow Java naming conventions for constants. Consider renaming it to 'REPORT_DATA_DELIMITER'.

Reply:
++